quick-start/examples/n4.min.js

```diff
@@ -0,0 +1,15 @@
+import { Scene, Circle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+// Any CSS valid color is supported (named, hex, rgb, hsl ...)
```

Comment:
```suggestion
// Any valid CSS color is supported (named, hex, rgb, hsl ...)
```
Put the adjective in front of the compound noun.